test/hotspot/jtreg/gc/g1/humongousObjects/objectGraphTest/GCTokens.java

```diff
@@ -33,7 +33,7 @@ private GCTokens() {
 
     public static final String WB_INITIATED_YOUNG_GC = "Young (Normal) (WhiteBox Initiated Young GC)";
     public static final String WB_INITIATED_MIXED_GC = "Young (Mixed) (WhiteBox Initiated Young GC)";
-    public static final String WB_INITIATED_CMC = "WhiteBox Initiated Concurrent Mark";
+    public static final String WB_INITIATED_CMC = "WhiteBox Initiated Run to Breakpoint";
```

Comment:
`WB_INITIATED_CMC = "WhiteBox Initiated Run to Concurrent GC Breakpoint"; ` as we indicate below that `String CMC = "Concurrent Mark)"`

Reply:
Sorry, I don't understand the question/comment?  The change to the string
reflects what is logged by the VM now that we're using GC breakpoints to
control the collection.  The logging being matched with CMC didn't change.

Reply:
ok, then it is fine.